http_service/app.py

```diff
@@ -110,13 +204,124 @@ def model_prediction(model_name, bug_id):
 
 
 @application.route("/<model_name>/predict/batch", methods=["POST"])
+@cross_origin()
 def batch_prediction(model_name):
+    """
+    ---
+    post:
+      description: >
+        Post a batch of bug ids to classify, answer either 200 if all bugs are
+        processed or 202 if at least one bug is not processed.
+        <br/><br/>
+        Starts by sending a batch of bugs ids like this:<br/>
+        ```
+        {"bugs": [123, 456]}
+        ```<br/><br>
+
+        You will likely get a 202 answer that indicates that no result is
+        available yet for any of the bug id you provided with the following
+        body:<br/>
+
+        ```
+        {"bugs": {"123": {ready: False}, "456": {ready: False}}}
+        ```<br/><br/>
+
+        Call back the same endpoint with the same bug ids a bit later, and you
+        will get the results.<br/><br/>
+
+        You might get the following output if some bugs are not available:
+        <br/>
+
+        ```
+        {"bugs": {"123": {"available": False}}}
+        ```<br/><br/>
+
+        And you will get the following output once the bugs are available:
+        <br/>
+        ```
+        {"bugs": {"456": {"extra_data": {}, "index": 0, "prob": [0], "suggestion": ""}}}
+        ```<br/><br/>
+
+        Please be aware that the results might be interleaved, so the
+        following output is valid:
```

Comment:
What did you mean here?

Reply:
I meant that for a batch, some of the bugs might be available, some of the bugs might still be processing and some of the bugs results might be ready

Reply:
Oh, OK, can you specify that?
Something like:
```
Please be aware that the results might be interleaved, so the following output, where a bug is returned and another one is still being processed, is valid:
```

Reply:
Ok done